htdocs/product/stock/class/mouvementstock.class.php

```diff
@@ -379,6 +388,17 @@ public function _create($user, $fk_product, $entrepot_id, $qty, $type, $price =
 		$movestock = 0;
 		if ($product->type != Product::TYPE_SERVICE || !empty($conf->global->STOCK_SUPPORTS_SERVICES)) $movestock = 1;
 
+		// check unicity for serial numbered equipments ( different for lots managed products)
```

Comment:
If we have a corruption of data and qty in stock is 10 and we want to fix to get 1 (because it is a unique serial) for example after a direct import or because we changed the status of batch from 1 to 2 later, this test will make not possible to fix situation.
Must be removed.

Reply:
ok i do it as for the previous, but we will have to have those tests elsewhere for dispatch, stock transfer or correction, bom, apis...